sdk/translation/azure-ai-translation-document/tests/test_model_updates.py

```diff
@@ -0,0 +1,253 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import datetime
+import functools
+from azure.ai.translation.document.models import DocumentStatus, FileFormatType, StatusSummary, TranslationGlossary, TranslationStatus
+from testcase import DocumentTranslationTest
+from preparer import (
+    DocumentTranslationPreparer,
+    DocumentTranslationClientPreparer as _DocumentTranslationClientPreparer,
+)
+from devtools_testutils import recorded_by_proxy
+from azure.ai.translation.document import DocumentTranslationClient, DocumentTranslationInput, TranslationTarget
+
+DocumentTranslationClientPreparer = functools.partial(_DocumentTranslationClientPreparer, DocumentTranslationClient)
+
+class TestModelUpdates(DocumentTranslationTest):
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    @recorded_by_proxy
+    def test_start_translation_details_model(self, **kwargs):
+        client = kwargs.pop("client")
+        variables = kwargs.pop("variables", {})
+
+        docs_count = 2
+        self._prepare_and_validate_start_translation_details(client, docs_count, wait=False, variables=variables)
+        return variables
+    
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    @recorded_by_proxy
+    def test_supported_formats(self, **kwargs):
+        client = kwargs.pop("client")
+        
+        # get supported glossary formats
+        supported_glossary_formats = client._get_supported_formats(type=FileFormatType.GLOSSARY)
```

Comment:
We don't need to test internal methods. Don't we already test the get_supported_* methods in existing tests?

Reply:
Yes, we do have test coverage for get_supported_document_format and get_supported_glossary_format methods, hence removed.